backend/backend.py

```diff
@@ -32,6 +33,20 @@ def __init__(self, width: int, height: int, publish_recording_status_func: Publi
         # Time between recording statuses, in seconds
         self.recording_status_interval = 60
 
+        # time that we last called self.send_camera_is_online
+        self.prev_camera_is_online_time = 0
+        # Time between recording statuses, in seconds
+        self.camera_is_online_interval = 60
+
+    @abstractmethod
+    def _is_camera_on(self) -> bool:
```

Comment:
Don't have to change it, but something about starting this method name with "is" annoys me lmao

Reply:
"is" is the convention for a function that just checks some condition and returns a boolean

Reply:
Ah yeah true. Didn't think about it like that 👍